authentication/README.md

```diff
@@ -0,0 +1,83 @@
+# Azure AI Content Safety Authentication
+
+Each request to an Azure Cognitive Service must include an authentication header. This header passes along a subscription key or authentication token, which is used to validate your subscription for a service or group of services.
+
+There are two kinds of authentication headers available for Azure AI Content Safety Service.
+| Header | Description |
+|--------|-------------|
+| Ocp-Apim-Subscription-Key | Use this header to authenticate with a subscription key for a specific service or a multi-service subscription key. |
+| Authorization | Use this header if you are using an access token. The steps to perform a token exchange are detailed in the following sections. The value provided follows this format: `Bearer <TOKEN>`. |
+
+## Authenticate with API key
+
+If you have created your Content Safety resource, you can go to azure portal, click into the resource, and find the keys under `Keys and Endpoint` tab.
+
+![get_api_key](image_get_key.png)
+
+If you are making http request to Restful API directly, you could set this value to `Ocp-Apim-Subscription-Key` in http header. In C# it may look like:
+
+```csharp
+HttpClient client = new();
+client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", "[Your API Key]");
+```
+
+If you are using Azure AI Content Safety SDK, you should create an `AzureKeyCredential` with this key, then construct the `ContentSafetyClient`. In dotnet SDK it may look like:
+
+```csharp
+string endpoint = "[Your endpoint]";
+string key = "[Your subscription key]";
+
+ContentSafetyClient client = new ContentSafetyClient(new Uri(endpoint), new AzureKeyCredential(key));
+```
+
+## Authenticate with Azure Active Directory
+
+While the keys provide a quick and easy path to start development, they fall short in more complex scenarios that require Azure role-based access control (Azure RBAC). In this scenario, you will need to authenticate with Azure Active Directory(AAD).
+
+**Currently we only support AAD authentication when calling Restful API by http directly, SDKs will support it in later versions.**
```

Comment:
Probably changed to "AAD authentication is supported when calling Azure AI Content Safety RESTful APIs over HTTP currently. It will also be supported in future versions of the Azure AI Content Safety SDK."